### DISCUSSION TITLE
Build Time Hooks / Extensions

### DISCUSSION THREAD

**Discussion by VinayaSathyanarayana** [keystonejs/keystone#5346] [Feature requests]
Build time hooks / extensions / plug-ins to cover use cases like

- [ ]  Generate OpenAPI Specs from Keystone List configuration

 

- [ ] Generate PlantUML Diagrams from prima schema

- [ ]  Generate sitemap

 .... Others

**Comment by bladey** [comment]
Thanks for the suggestions @VinayaSathyanarayana.
